sample/ingress-controller-tls/README.md

```diff
@@ -1,119 +1,9 @@
-
-```yaml
-      volumes:
-        - name: secrets-store-inline
-          csi:
-            driver: secrets-store.csi.k8s.io
-            readOnly: true
-            volumeAttributes:
-              secretProviderClass: "azure-tls"
-            nodePublishSecretRef:
-              name: secrets-store-creds
-```
-
-```bash
-kubectl apply -f sample/ingress-controller-tls/deployment-app-one.yaml -n ingress-test
-kubectl apply -f sample/ingress-controller-tls/deployment-app-two.yaml -n ingress-test
-
-```
-
-# Check for the Kubernetes Secret created by the CSI driver
-```bash
-kubectl get secret -n ingress-test
-
-NAME                                             TYPE                                  DATA   AGE
-ingress-tls-csi                                  kubernetes.io/tls                     2      1m34s
-```
-
-# Deploy an Ingress Resource referencing the Secret created by the CSI driver
-
-> NOTE: The ingress resource references the Kubernetes secret `ingress-tls-csi` created by the CSI driver as a result of the app creation.
-
-```yaml
-tls:
-  - hosts:
-    - demo.test.com
-    secretName: ingress-tls-csi
-```
-
-```bash
-kubectl apply -f sample/ingress-controller-tls/ingress.yaml -n ingress-test
-```
-
-# Get the External IP of the Ingress Controller
-
-```bash
- kubectl get service -l app=nginx-ingress --namespace ingress-test 
-NAME                                       TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                      AGE
-nginx-ingress-1588032400-controller        LoadBalancer   10.0.255.157   52.xx.xx.xx   80:31293/TCP,443:31265/TCP   19m
-nginx-ingress-1588032400-default-backend   ClusterIP      10.0.223.214   <none>           80/TCP                       19m
-```
-
-# Test Ingress with TLS
-Using `curl` to verify ingress configuration using TLS. 
-Replace the public IP with the external IP of the ingress controller service from the previous step.  
+The Secrets Store CSI Driver can be used to enable applications to work with NGINX Ingress Controller with TLS stored in an External Secrets Store. 
+For more information on securing an Ingress with TLS, refer to: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
 
-```bash
-curl -v -k --resolve demo.test.com:443:52.xx.xx.xx https://demo.test.com
+Checkout provider samples on how to get started -
 
-# You should see the following in your output
-*  subject: CN=demo.test.com; O=ingress-tls
-*  start date: Apr 15 04:23:46 2020 GMT
-*  expire date: Apr 15 04:23:46 2021 GMT
-*  issuer: CN=demo.test.com; O=ingress-tls
-*  SSL certificate verify result: self signed certificate (18), continuing anyway.
-```
+- [Using Secrets Store CSI and Azure Key Vault Provider](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/sample/ingress-controller/README.md)
```

Comment:
The folder here should be consistent across all repos. how about `sample/ingress-controller-tls`?